From 2eefcf6d997b907f2d6c66b430474966ca514100 Mon Sep 17 00:00:00 2001
From: Albin B <albin.bala.krishnan@intel.com>
Date: Tue, 24 Nov 2015 11:15:07 +0530
Subject: [PATCH 03/12] [USBC_PD]: Correct the Type-C Sink Wait Cap timer value

The Sink Wait Cap timer value is different between Type-C connector
and all other connectors. In the existing code uses the SinkWaitCap
timer value is for non Type-C connectors. So corrected the Type-C
connectors SinkWaitCap timer value.

Change-Id: I2d7179c85a81ae081db30757018a3eb8e2e16587
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-10629
Signed-off-by: Albin B <albin.bala.krishnan@intel.com>
Reviewed-on: https://android.intel.com:443/439715
(cherry picked from commit e9788c67f87b82f3acd5d294caed4ecc37cd7846)
Reviewed-on: https://android.intel.com:443/443158
---
 drivers/usb/typec/pd/sink_port_pe.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/typec/pd/sink_port_pe.h b/drivers/usb/typec/pd/sink_port_pe.h
index 35b9e87..4e5be89 100644
--- a/drivers/usb/typec/pd/sink_port_pe.h
+++ b/drivers/usb/typec/pd/sink_port_pe.h
@@ -48,7 +48,7 @@
 #define REQ_DOBJ_MAX_OPERATING_CUR	(0x3FF << REQ_DOBJ_MAX_OP_CUR_SHIFT)
 
 #define TYPEC_SENDER_RESPONSE_TIMER	30 /* min: 24mSec; max: 30mSec */
-#define TYPEC_SINK_WAIT_CAP_TIMER	2500 /* min 2.1Sec; max: 2.5Sec */
+#define TYPEC_SINK_WAIT_CAP_TIMER	620 /* min 310mSec; max: 620mSec */
 #define TYPEC_NO_RESPONSE_TIMER		5500 /* min 4.5Sec; max: 5.5Sec */
 #define TYPEC_PS_TRANSITION_TIMER	550 /* min 450mSec; max: 550mSec */
 #define TYPEC_SINK_ACTIVITY_TIMER	150 /* min 120mSec; max: 150mSec */
-- 
1.9.1

